packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -603,31 +592,27 @@ export class MaterialPanel extends ConnectedLitElement {
     return html`
   <me-expandable-tab tabName="Base Color">
     <div slot="content">
-      <me-section-row label="Factor">
-        <div class="TexturePickerContainer">
-          <mwc-icon-button class="RevertButton" id="revert-base-color-factor" icon="undo"
-            title="Revert to original base color factor"
-            @click=${this.revertBaseColorFactor}></mwc-icon-button>
+      <me-card title="Factor" functionId="revert-base-color-factor" functionTitle="Revert to original base color factor"
+        .undoFunction=${this.revertBaseColorFactor.bind(this)}>
+        <div slot="content">
           <me-color-picker id="base-color-picker"
           selectedColorHex=${selectedColorHex} @change=${
         this.onBaseColorChange}>
           </me-color-picker>
         </div>
-      </me-section-row>
-      <me-section-row label="Texture">
-        <div class="TexturePickerContainer">
-          <mwc-icon-button class="RevertButton" id="revert-base-color-texture" icon="undo"
-          title="Revert to original base color texture"
-            @click=${this.revertBaseColorTexture}></mwc-icon-button>
+      </me-card>
+      <me-card title="Texture" functionId="revert-base-color-texture" functionTitle="Revert to original base color texture"
+        .undoFunction=${this.revertBaseColorTexture.bind(this)}>
```

Comment:
Might be better to call this `revertFunction`, as undo usually means "go one step back", and we may well implement a real undo before long.

Reply:
good call, I was basing those off of the icon, but you're right

Reply:
done